lab_1/main.py

```diff
@@ -11,7 +11,19 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    if type(text) == str:
```

Comment:
it's better to check arguments first. If they're incorrect, return None

So, move line 25 to the beginning